src/modules/MediaCarousel.php

```diff
@@ -15,7 +15,10 @@ public function get_items_field() {
 	 * @return array
 	 */
 	public function get_fields() {
-		return [ 'image_link_to' => [ 'url' ] ];
+		return [ 
```

Comment:
did it work without field_id?

Reply:
Out of the box, you can't use `field` and `field_id` with the compatibility class.

This REVERTED commit shows what I needed to do to use `field_id`: 
https://github.com/OnTheGoSystems/wpml-page-builders-elementor/commit/87e6779dfa2d9694aca69d0e305dcc8b066da23e

^I am not sure how this would affect other widgets.

Reply:
Well, if it is going to affect other widgets, I think this can wait for the new wpml-config.xml file.

Reply:
I think there's a bit of confusion here. I REVERTED the commit that would affect other widgets.

This MR simply registers the video field in MediaCarousel.php and adjusts the Test class accordingly.

Reply:
😅  yes, sorry it's my fault.

Reply:
Thanks, I CAN be confusing 😛